README.md

```diff
@@ -15,6 +15,39 @@ yarn add quiq-chat
 
 ## Usage
 
+### Using `QuiqChatClient`
+
+The default export of `quiq-chat` is the `QuiqChatClient` class which will fetch information about the current webchat, initialize a websocket connection, and allow you to register callbacks so that you can keep your app's UI in sync with the webchat state.
+
+All the functions to register callbacks return the `QuiqChatClient` object so that you can chain them together. You also need to call `start()` to connect to Quiq Messaging. The `start` method returns a promise that resolves to the `QuiqChatClient`, so you can add a callback that will be executed after the connection is opened;
+
+```javascript
+import QuiqChatClient from 'quiq-chat';
+
+const client = new QuiqChatClient()
+  .onNewMessages(messages => {
```

Comment:
Oops, yeah, I forgot. I'll probably just put it in the documentation section at the bottom though